include/arrayfire.h

```diff
@@ -230,6 +230,7 @@
 #include "af/exception.h"
 #include "af/features.h"
 #include "af/gfor.h"
+#include "af/graphics.h"
```

Comment:
I think this is fine because we are compiling the functions even if they are not supported.